### PR TITLE
Add clang format file

### PR DESCRIPTION
**Pull request by Vicx95**

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Language: Cpp
+BasedOnStyle: Google
+UseTab: Never
+IndentWidth: 4
+TabWidth: 4
+AllowShortIfStatementsOnASingleLine: Never
+IndentCaseLabels: false
+ColumnLimit: 0
```
Comment (lukasz126) [inline .clang-format:8]: maybe add limit e.g. 120?

File: .clang-format [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+AccessModifierOffset: -4
```
Comment (lukasz126) [inline .clang-format:9]: add EoF

Comment (lukasz126) [inline .clang-format:9]: maybe here add ignored files e.g:
Ignore:
  - 'communicator/mainwindow.h'

File: .clang-format (hunk 1 above)
Comment (Vicx95) [reply .clang-format:8]: done

File: .clang-format (hunk 2 above)
Comment (Vicx95) [reply .clang-format:9]: done

Comment (Vicx95) [reply .clang-format:9]: why?

**Review by iprice73 — APPROVED** [review]

**Comment by ziobron** [comment]
Thanks @iprice73 for Code Review.
🏅 1 XP granted

**Review by lukasz126 — APPROVED** [review]

**Review by lukasz126 — APPROVED** [review]

**Comment by ziobron** [comment]
Your PR was merged!
🏅 2 XP granted. Thanks @Vicx95 for making course materials better!

**Pull request merged**